oci/prometheus/builds.yaml

```diff
@@ -0,0 +1,6 @@
+version: 1
+
+images:
+  - source: "canonical/prometheus-rock"
```

Comment:
@cjdcordeiro confirmed that this is indeed a list, and we can have multiple workload and/or ubuntu versions.

Shall we try it out?

Reply:
How would we approach the different tags in this case, @cjdcordeiro ?

Reply:
@sed-i yes indeed that's the purpose of that list.

@athos-ribeiro tags are achieved via <track>_<edge>. For now, only the <track = version-base>_<risk> channels are allowed as tags